src/pages/LoginPage.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import LoginForm from '../components/LoginForm'
+import { address } from "ip";
```

Comment:
You are not using ';' after import functions and everywhere in project. You should write code in one style. If you are not planning to separate each function in your code, remove it.